dev-docs/bidders/deepintent.md

```diff
@@ -8,6 +8,8 @@ biddercode: deepintent
 media_types: banner, video
 tcfeu_supported: true
 usp_supported: true
+coppa_supported: true
+gpp_supported: true
```

Comment:
This field is superseded by `gpp_sids`. See https://docs.prebid.org/dev-docs/bidder-adaptor.html#submitting-your-adapter for supported values

Reply:
thanks, updated the doc. A test case seems to be failing on the implementation PR but that is in a different adapter altogether. Not sure how I can fix that. Could you please help out ?